### PR TITLE
DR-560 Fix all LoggerFactory uses to use the .class interface instead of text strings

### PR DESCRIPTION
**Pull request by myessail**

### CODE REVIEW

File: src/main/java/bio/terra/service/upgrade/Migrate.java [hunk 1]
```diff
@@ -32,7 +32,7 @@
  */
 @Component
 public class Migrate {
-  private static final Logger logger = LoggerFactory.getLogger("bio.terra.service.upgrade");
+  private static final Logger logger = LoggerFactory.getLogger(UpgradeService.class);
```
Comment (nmalfroy) [inline src/main/java/bio/terra/service/upgrade/Migrate.java]: Wouldn't this be the Migrate class?

File: src/main/java/bio/terra/service/dataset/flight/ingest/IngestCleanupStep.java [hunk 2]
```diff
@@ -10,7 +10,7 @@
 import org.slf4j.LoggerFactory;
 
 public class IngestCleanupStep implements Step {
-  private Logger logger = LoggerFactory.getLogger("bio.terra.service.dataset.flight.ingest");
+  private Logger logger = LoggerFactory.getLogger(DatasetIngestFlight.class);
```
Comment (nmalfroy) [inline src/main/java/bio/terra/service/dataset/flight/ingest/IngestCleanupStep.java]: Would this not be the IngestCleanupStep class?

**Review by nmalfroy — CHANGES_REQUESTED** [review]
I think some of the classes might be wrong

**Comment by pshapiro4broad** [comment]
I'm not a fan of everything (or most things) in Lombok, but its `@Log` annotation would avoid this boilerplate, and we wouldn't have to worry about making the `getLogger()` call match the class.  https://projectlombok.org/features/log

In DSP, the policy team is using this feature of Lombok in their project.

**Review by snf2ye — APPROVED** [review]

**Review by fboulnois — APPROVED** [review]

**Review by nmalfroy — APPROVED** [review]
thanks for making the changes!

**Pull request merged**